WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -1243,11 +1243,19 @@ protected void showInvalidUsernameOrPasswordDialog() {
         bundle.putString(SignInDialogFragment.ARG_OPEN_URL_PARAM, getForgotPasswordURL());
         bundle.putString(ENTERED_URL_KEY, EditTextUtils.getText(mUrlEditText));
         bundle.putString(ENTERED_USERNAME_KEY, EditTextUtils.getText(mUsernameEditText));
+        passHelpshiftErrorOriginTag(bundle);
         nuxAlert.setArguments(bundle);
         ft.add(nuxAlert, "alert");
         ft.commitAllowingStateLoss();
     }
 
+    protected void passHelpshiftErrorOriginTag(Bundle bundle) {
+        bundle.putSerializable(HelpshiftHelper.ORIGIN_KEY,
+                isWPComLogin() ? Tag.ORIGIN_LOGIN_SCREEN_ERROR_WPCOM
+                        : (isJetpackAuth() ? Tag.ORIGIN_LOGIN_SCREEN_ERROR_JETPACK
```

Comment:
That was actually my understanding anyway (after talking with @rachelmcr ) that we want to pinpoint that case where the user logs in to Jetpack while trying to view stats so, you seem to confirm that using `isJetpackAuth()` is the appropriate thing to do then.

> but this should not be used for stats

Hmm, not following you here @maxme , can you elaborate?

> Adding a comment on top of that code would be great.

Good idea, will do!